modules/auxiliary/scanner/msmail/exchange_enum.go

```diff
@@ -90,18 +89,12 @@ func o365enum(ip string, emaillist []string, threads int) []string {
 			for email := range queue {
 				responseCode := msmail.WebRequestBasicAuth(URI, email, pass, tr)
 				if strings.Contains(email, "@") && responseCode == 401 {
-					mux.Lock()
 					module.LogInfo("[+]  " + email + " - 401")
 					validemails = append(validemails, email)
-					mux.Unlock()
 				} else if strings.Contains(email, "@") && responseCode == 404 {
-					mux.Lock()
-					module.LogInfo(fmt.Sprintf("[-]  %s - %d \n", email, responseCode))
-					mux.Unlock()
+					module.LogInfo(fmt.Sprintf("[-]  %s - %d", email, responseCode))
```

Comment:
You still left `[-]` which gets doubled up. This should say 'LogError' or whatever yields `[-]`